examples/r-api/README.md

```diff
@@ -45,6 +49,12 @@ function(bedrooms,year,res){
     return(list(error = "Please enter BedroomAbvGr between 0 and 8. Enter YearBuilt between 1872 and 2100."))
   }  
 }
+#* Redirect to Docs
+#* @get /
+function(res) {
+  res$status <- 303 # redirect
```

Comment:
How did you choose 303? I think 301 would be more normal (docs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections)

Reply:
I choose since it stands for redirecting to a different resource https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303 . But if 301 makes more sense here, I can replace with that ,

Reply:
I just checked with Alex and he says 302 is probably best.

Reply:
okie cool .. Will change in the other pr as well